src/logger/loggerApi.c

```diff
@@ -1237,42 +1238,82 @@ int api_getGpsConfig(struct Serial *serial, const jsmntok_t *json)
     json_int(serial, "alt", gpsCfg->altitude.sampleRate != SAMPLE_DISABLED, 1);
     json_int(serial, "sats", gpsCfg->satellites.sampleRate != SAMPLE_DISABLED, 1);
     json_int(serial, "qual", gpsCfg->quality.sampleRate != SAMPLE_DISABLED, 1);
-    json_int(serial, "dop", gpsCfg->DOP.sampleRate != SAMPLE_DISABLED, 0);
+    json_int(serial, "dop", gpsCfg->DOP.sampleRate != SAMPLE_DISABLED, 1);
+
+    json_objStartString(serial, "units");
+    json_string(serial, "altitude", gpsCfg->altitude.units, 1);
```

Comment:
recommend making these distance and altitude keys consistent with the other keys in this message.